### PR TITLE
doc/cephfs: add note to remind user about snap-schedule during volume delete

### PR DESCRIPTION
**Pull request by mchangir**
doc/cephfs: add note to remind user about snap-schedule during volume delete #60362 

Signed-off-by: Milind Changire <mchangir@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

File: doc/cephfs/fs-volumes.rst [hunk 1]
```diff
@@ -72,6 +72,14 @@ tries to remove MDS daemons using the enabled Ceph Manager orchestrator module.
    being used. See https://tracker.ceph.com/issues/49605#note-5 for more
    details.
 
+.. note:: If the snap-schedule Ceph Manager module is being used for a volume
+   and the volume is deleted, then the snap-schedule Ceph Manager module will
+   continue to hold references to the old pools. This will lead to the
+   snap-schedule Ceph Manager module faulting and logging errors. To remedy
+   this scenario, we recommend that the snap-schedule Ceph Manager module
```
Comment (joscollin) [inline doc/cephfs/fs-volumes.rst:79]: s/we/it is

File: doc/cephfs/fs-volumes.rst [hunk 2]
```diff
@@ -72,6 +72,14 @@ tries to remove MDS daemons using the enabled Ceph Manager orchestrator module.
...
+   be restarted after volume deletion. If the faults still persist, then we
+   recommend restarting `ceph-mgr`.
+
```
Comment (joscollin) [inline doc/cephfs/fs-volumes.rst:82]: same here too^

File: doc/cephfs/fs-volumes.rst (hunk 1 above)
Comment (mchangir) [reply doc/cephfs/fs-volumes.rst:79]: I had **it is** before I noticed that elsewhere it was **we**.
Anthony too didn't raise any concerns about the language.

Comment (anthonyeleven) [reply doc/cephfs/fs-volumes.rst:79]: Zac has kinda encouraged me to follow up with my own wordsmithing PRs after merge.

**Pull request merged**